src/main/java/org/jabref/gui/externalfiles/UnlinkedFilesDialogViewModel.java

```diff
@@ -145,9 +145,11 @@ public void startSearch() {
     }
 
     public void startImport() {
+        Path directory = this.getSearchDirectory();
         List<Path> fileList = checkedFileListProperty.stream()
                                                      .map(item -> item.getValue().getPath())
                                                      .filter(path -> path.toFile().isFile())
+                                                     .map(path -> directory.relativize(path))
```

Comment:
Can you describe why this should change to relative paths for importing? I see only an unnecessary step costing computation time, but maybe there is something i dont see...

Reply:
Representing relative paths in the File column was one of the points mentioned in the issue. However, to create these relative paths, it was necessary to obtain the absolute path of the directory and the selected files. That is why I chose this approach to solve the problem.

Reply:
Ok, I see.